docs/DeveloperGuide.md

```diff
@@ -1,34 +1,503 @@
+<center><i>Figure 12: Exception Classes</i></center>
+
+Given below are the instructions to test the app manually.
+
+### 6.1 Launch and Shutdown (Felix)
+
+1. Initial Launch
+   1. Verify that you have Java 11 or above installed in your computer by running `java --version`.
+   2. Download the latest `trippie.jar` from [here](https://github.com/AY2021S1-CS2113T-W11-2/tp/releases/tag/v2.0).
+   3. Copy the file to a folder where you want to run it from.
+   4. Using a command line tool, navigate to the path of `trippie.jar`, by entering `cd path/to/file`
+       * If your operating system is Windows, use Command Prompt.
+       * If your operating system is MacOS, use Terminal.
+   6. Enter `java -jar trippie.jar` in your command line tool and press enter.
+   7. If the setup is correct, you should see a welcome message. Then, it is now ready to go.
+   8. Create your first trip by entering `new trip` to the command line. Then, enter your trip name, start date, exhange rate, currency, and budget.
+   9. Add your first place using `add`. Then buy your first item with `buy`.
+   10. Try listing your places by entering `list /p` and your expenses with `list /e`.
+2. Shutdown
+   1. `exit` to exit Trippie.
+   2. Your Trippie files will be automatically saved in ./trippie_data!
+
+### 6.2 Step by Step Guide
+
+#### Creating a new trip
+
+1. Creates a new trip and sets it as the current trip.
+* Test Case:  Enter the command `new trip`. Trippie will prompt you to enter the trip name. Try inputting an invalid file name, like `A trip of 01/01`. 
+
+* Expected: Trippie will ask you to reenter the name.
+
+```
+Enter your new trip's name:A trip of 01/01
+New trip should not contain invalid characters like <>:"/\|?*!
+Enter your new trip's name:
+```
+
+#### Loading an existing trip
+
+* Test case: Create a new trip. Enter the command `load trip`. Trippie will prompt you to enter a trip index. Which you can enter correspondingly.
+* Expected: 
+
+```
+_________________________________________________________________________
+Here are your existing trips.
+1. 0 Days - test
+
+Which one do you want to load? Enter the index:1
+_________________________________________________________________________
+```
+
+#### Viewing help
+
+* Test case: Enter the command `help`.
+* Expected: Trippie will show you a table of commands.
+
+#### Adding a place
+
+* Test case: Key in `add /n test place /d 3 /t 0000 to 0100` into the CLI.
+* Expected:
+```
+Got it. I've added this place:
+0000 - 0100 test place
+Now you have 1 place in the list.
+```
+
+#### Listing all places
+
+* Test case: Enter the command `list /p`. Trippie will then list out the places that you have added into the trip.
+* Expected:
+```
+_________________________________________________________________________
+DAY 3: (2011-01-03)
+[1] 0000 - 0100 test place
+_________________________________________________________________________
+```
+
+#### Searching all places
```

Comment:
There are some missing output for the 2 functions below. Kindly remember to fill it up before submission